src/constants/property-metadata.constants.ts

```diff
@@ -431,6 +431,1468 @@ export const propertyMetadata = {
+      example: "2019-05-07",
+    },
+    hourWeeklyTestSummaryDto: {
+      fieldLabels: {
+        label: "Test Hour",
+        value: "testHour",
+      },
+      description: "The hour corresponding to the completion of the test.",
+      example: "15",
+    },
+    minuteWeeklyTestSummaryDto: {
+      fieldLabels: {
+        label: "Test Minute",
+        value: "testMinute",
+      },
+      description: "The minute corresponding to the completion of the test.",
+      example: "0",
+    },
+    weeklySystemIntegrityId: {
+      fieldLabels: {
+        label: "Weekly System Integrity ID",
+        value: "id",
+      },
+      description: "Unique identifier of a weekly system integrity record.",
+      example: "PAULDESKTO-43BFEC874EE54620875D85BAE00FA83D",
+    },
+    weeklyTestSummaryIdWeeklySystemIntegrityDto: {
+      fieldLabels: {
+        label: "Weekly Test Summary ID",
+        value: "weeklyTestSumId",
+      },
+      description: "Unique identifier of a weekly test summary record. ",
+      example: "PAULDESKTO-98968A0A86D449F7A53F3176BA9B8B99",
+    },
+    apsIndicator: {
+      fieldLabels: {
+        label: "APS Indicator",
+        value: "apsIndicator",
+      },
+      description:
+        'If the system integrity error (SIE) is determined using the standard performance specification, "0" is reported. If the alternative performance specification (APS) is used to pass the test, "1" is reported.',
+      example: "0",
+    },
+    systemIntegrityError: {
+      fieldLabels: {
+        label: "System Integrity Error",
+        value: "systemIntegrityError",
+      },
+      description:
+        "The results of the single-point system integrity check as a percentage of the reference gas value or, if necessary, as the absolute value of the difference between the reference and measured values.  Reported as SystemIntegrityError in Weekly System Integrity Data record.",
+      example: "7.4",
+    },
+    gasLevelCode: {
+      fieldLabels: {
+        label: "Gas Level Code",
+        value: "gasLevelCode",
+      },
+      description:
+        "Identifies whether the standard used is a high- or mid-level NIST-traceable source of oxidized Hg. Reported as GasLevelCode in the Weekly System Integrity Data record.",
+      example: "HIGH",
+    },
+    referenceValue: {
+      fieldLabels: {
+        label: "Reference Value",
+        value: "referenceValue",
+      },
+      description:
+        "Value of the calibration standard used for the injection (ug/scm).  Reported as ReferenceValue in Weekly System Integrity Data record.",
+      example: "2.7",
+    },
+    measuredValue: {
+      fieldLabels: {
+        label: "Measured Value",
+        value: "measuredValue",
+      },
+      description:
+        "Value measured by instrument for the calibration standard injection (ug/scm).  Reported as MeasuredValue in Weekly System Integrity Data record.",
+      example: "2.5",
+    },
+  },
```

Comment:
Values from the master list I did not see:

![2022-08-10 12_50_29-ECMPS - EM Master Data Labels List (1) xlsx - Excel](https://user-images.githubusercontent.com/3924176/183982002-4f45f946-e582-46a2-a308-70254f5e7e26.png)

* calcOnlineOfflineIndicator
* calcTestResultCode
* calcUpscaleAPSInd
* calcUpscaleCalibrationError
* calcZeroAPSIndicator
* calcZeroCalibrationError
* spikeTrapHg
* volumetricFlowRate

























































*